main.go

```diff
@@ -49,17 +50,25 @@ func main() {
 			EnvVar: "DOCKER_HOST",
 		},
 		cli.IntFlag{
-			Name:  "interval, i",
-			Usage: "poll interval (in seconds)",
-			Value: 300,
+			Name:   "interval, i",
+			Usage:  "poll interval (in seconds)",
+			Value:  300,
+			EnvVar: "WATCHTOWER_POLL_INTERVAL",
 		},
 		cli.BoolFlag{
-			Name:  "no-pull",
-			Usage: "do not pull new images",
+			Name:   "no-pull",
+			Usage:  "do not pull new images",
+			EnvVar: "WATCHTOWER_NO_PULL",
 		},
 		cli.BoolFlag{
-			Name:  "cleanup",
-			Usage: "remove old images after updating",
+			Name:   "no-restart",
+			Usage:  "do not restart containers",
+			EnvVar: "WATCHTOWER_NO_PULL",
```

Comment:
This environment variable name is wrong; probably you wanted something like `WATCHTOWER_NO_RESTART`.